recipe/meta.yaml

```diff
@@ -34,11 +34,12 @@ requirements:
     - gdk-pixbuf
     - gettext  # [osx]
     - glib
-    - libgraphene
     - harfbuzz
+    - jpeg  # [win]
+    - libgraphene
     - libcups  # [linux]
     - libiconv  # [win]
-    - pango
+    - pango >=1.50
```

Comment:
Shouldn't this be set in conda_build_config.yaml file instead?

Is there going to be a conda-forge wide migration to Pango 1.50?

Reply:
You're probably right about the `conda_build_config.yaml`, yes. To be honest I don't have a good feeling for what should go in that file vs. what should go in `meta.yaml`.

As for project-wide migration, I know of no reason not to do one! Part of my reason for pushing this update was to double-check that a package like gtk4 actually builds with the new version, and it looks like that part is OK.